src/controllers/list.js

```diff
@@ -791,21 +791,37 @@ class ItemsView {
 
                 const itemType = item ? item.Type : null;
 
-                if (itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel') {
+                if ((itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel')
+                    // Folder, Playlist views
+                    && itemType !== 'UserView'
+                    // Only Photo (homevideos) CollectionFolders are supported
+                    && !(itemType === 'CollectionFolder' && item?.CollectionType !== 'homevideos')
+                ) {
+                    // Show Play All buttons
                     hideOrShowAll(view.querySelectorAll('.btnPlay'), false);
                 } else {
+                    // Hide Play All buttons
                     hideOrShowAll(view.querySelectorAll('.btnPlay'), true);
                 }
 
-                if (itemType === 'MusicGenre' || params.type !== 'Programs' && params.type !== 'nextup' && itemType !== 'Channel') {
+                if ((itemType === 'MusicGenre' || params.type !== 'Programs' && params.type !== 'nextup' && itemType !== 'Channel')
+                    // Folder, Playlist views
+                    && itemType !== 'UserView'
+                    // Only Photo (homevideos) CollectionFolders are supported
+                    && !(itemType === 'CollectionFolder' && item?.CollectionType !== 'homevideos')
```

Comment:
Wouldn't that make it easier to perceive?
```suggestion
                    && (itemType !== 'CollectionFolder' || item?.CollectionType === 'homevideos')
```